config/game_options.txt

```diff
@@ -475,7 +475,7 @@ OVERFLOW_CAP -1
 #STARLIGHT
 
 ## Uncomment to bring back old grey suit assistants instead of the now default rainbow colored assistants.
-#GREY_ASSISTANTS
+GREY_ASSISTANTS
```

Comment:
Forgot to mention this. Brings back the old greyshirts instead of the random shirt colour. Grey-tide, station-wide